From: Thomas Renninger <trenn@suse.de>
Subject: Cleanup: No need to suppress possible errno anymore
References: 
Patch-Mainline: 
Git-commit: cb6f53ead8c1af3b0d984015638acbc46139e5ba


Since has_preferred_many is only called in numa_init
again (renamed to set_kernel_abi) there is no need
to suppress errno values. errno is set to zero at
end of numa_init anyway.

Signed-off-by: Thomas Renninger <trenn@suse.com>


Signed-off-by:  <trenn@suse.de>
diff --git a/libnuma.c b/libnuma.c
index 29349ee..2b515d0 100644
--- a/libnuma.c
+++ b/libnuma.c
@@ -627,9 +627,6 @@ set_kernel_abi(void)
 {
 	int oldp;
 	struct bitmask *bmp, *tmp;
-	int old_errno;
-
-	old_errno = errno;
 
 	bmp = numa_allocate_nodemask();
 	tmp = numa_get_mems_allowed();
@@ -648,7 +645,6 @@ set_kernel_abi(void)
 out:
 	numa_bitmask_free(tmp);
 	numa_bitmask_free(bmp);
-	errno = old_errno;
 }
 
 /*
